src/v2/handlers/lex-intent/index.js

```diff
@@ -0,0 +1,66 @@
+import {
+  LexModelsV2Client,
+  CreateIntentCommand,
+  DeleteIntentCommand,
+  UpdateIntentCommand
+} from "@aws-sdk/client-lex-models-v2";
+const logger = process.env.TEST ? { info: (c) => { } } : console;
+const client = new LexModelsV2Client({
+  region: process.env.REGION || "us-east-1",
+  logger
+});
+
+
+const handler = async (event, context) => {
+  try {
+    logger.info(event);
+    const params = JSON.parse(event.ResourceProperties.props);
+    let response = {};
```

Comment:
do we want to use the same pattern being used in other handlers with scoping on the switch and const response = in the blocks? I think you're fine, but maybe just for the sake of consistency.

Reply:
Yes, definitely!